external/fv3fit/tests/training/test_autoencoder.py

```diff
@@ -99,8 +58,8 @@ def test_autoencoder(tmpdir):
     os.chdir(tmpdir)
     # need a larger nx, ny for the sample data here since we're training
```

Comment:
Similar to previous request, I find this test lacking an overarching description of the data problem it's trying to capture with an autoencoder, i.e., it's necessary to get into the weeds of the code, or to take the time to run the test locally with the plotting uncommented, to understand what it's trying to do. Given the tricky nature of this test to write I think it's fair to have a verbose docstring describing where it landed.

Reply:
Will add some docstrings/comments.

With these settings the autoencoder isn't too bad. It's only a little slower than the other training tests.

Reply:
Updated. It didn't help that some of the previous docstrings/comments were false.